task/ecosystem-cert-preflight-checks/0.2/ecosystem-cert-preflight-checks.yaml

```diff
@@ -76,9 +76,21 @@ spec:
           exit 0
         fi
 
+        # If the image URL points to a manifest list (a multi-arch image), check the labels on any of the child
+        # images (don't fail in the case where the list does not include an image for the arch of the system
+        # where this pipeline is running).
+        _INSPECT_OVERRIDE_IMAGE_ARCH=
+        _INSPECT_OVERRIDE_IMAGE_OS=
+        _RAW_IMAGE_MANIFEST=$(skopeo inspect --raw "docker://${PARAM_IMAGE_URL}")
```

Comment:
Forgot one thing: would like to see an `echo` log line emitted just before this call so that, should we see a skopeo error in the log, we can differentiate between _this_ call and the call that comes later.